Source/Core/VideoCommon/VertexLoaderManager.cpp

```diff
@@ -425,24 +441,33 @@ int RunVertices(int vtx_attr_group, OpcodeDecoder::Primitive primitive, int coun
                           primitive < OpcodeDecoder::Primitive::GX_DRAW_LINES);
 
     const int stride = loader->m_native_vtx_decl.stride;
-    DataReader dst = g_vertex_manager->PrepareForAdditionalData(primitive, count, stride,
-                                                                cullall || can_cpu_cull);
+    do
+    {
+      const int max_vertices = 16380;  // Max is 16383, but 16380 is divisible by both 4 and 3
+      int run = CanSplit(primitive) && count > max_vertices ? max_vertices : count;
```

Comment:
```suggestion
      const int vertices_available = CanSplit(primitive) && count > max_vertices ? max_vertices : count;
```

Reply:
If you ask me, vertices_available sounds like the number of vertices that are sitting in memory available to be processed.  Which is the number currently in `count`.  This number is just the number we're going to process in this run through the loop so that the gpu draw generating code doesn't get overloaded.

Reply:
`vertices_to_process`?  Or maybe we leave this one as is.